hack/dashboard/assets/kepler/dashboard.json

```diff
@@ -102,7 +102,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "topk(10, sum by (container_namespace) (\n (increase(kepler_container_package_joules_total{container_namespace=~\".*\"}[1h])\n *$watt_per_second_to_kWh\n ) *\n (count_over_time(kepler_container_package_joules_total{container_namespace=~\".*\"}[24h])/\n count_over_time(kepler_container_package_joules_total{container_namespace=~\".*\"}[1h])\n )\n)\n+\nsum by (container_namespace) (\n (increase(kepler_container_dram_joules_total{container_namespace=~\".*\"}[1h])\n *$watt_per_second_to_kWh\n ) *\n (count_over_time(kepler_container_dram_joules_total{container_namespace=~\".*\"}[24h])/\n count_over_time(kepler_container_dram_joules_total{container_namespace=~\".*\"}[1h])\n )\n))",
+          "expr": "topk(10, sum by (container_namespace) (\n  increase(\n      (kepler_container_joules_total{}[24h:1m])\n  )\n) * $watt_per_second_to_kWh)",
```

Comment:
@vprashar2929 Sorry, but could this please wait until the prometheus-rules are also deployed? We can make use of the prometheus rules in that case which I find to be way faster.

Reply:
Absolutely 😊